usecase/character.go

```diff
@@ -0,0 +1,42 @@
+package usecase
+
+import "github.com/carRub/academy-go-q12021/model"
+
+// CharacterService interface to define the use cases applicable to this application/project
+type CharacterService interface {
+	GetCharacters() ([]model.Character, error)
+	GetCharacterByID(id int) (*model.Character, error)
+}
+
+// CharacterInteractor defines the use case fields
+type CharacterInteractor struct {
+	Service CharacterService
+}
+
+// NewCharacterInteractor generates a new character usecase
+func NewCharacterInteractor (s CharacterService) (*CharacterInteractor, error) {
+
+	return &CharacterInteractor{s}, nil
```

Comment:
As a recommendation, you should add the struct field names so you could easily identify the struct fields.

Also, get rid of leading empty lines inside functions.

```suggestion
	return CharacterController{
		characterInteractor: ci,
	}